packages/vision/src/helpers.js

```diff
@@ -74,11 +74,12 @@ var coerceImage = (image, callback) => {
  *   asking for the single feature annotation.
  */
 var _createSingleFeatureMethod = featureValue => {
-  return function(image, options) {
-    return this.annotateImage({
+  return function(image, options = {}, callOptions = undefined) {
```

Comment:
`callOptions` seem to have no effect on the method. I believe this runs into the issue I mentioned here: https://github.com/GoogleCloudPlatform/google-cloud-node/issues/2553#issuecomment-324385621 about consecutive, optional arguments. For example, if I call `cropHints({image}, {callOptions})`, my `callOptions` are going to be misunderstood for `options`, and end up extending the raw API request.

Side note, I don't think Node v4 supports default parameters.